src/main/java/seedu/stockpal/commands/HelpCommand.java

```diff
@@ -1,13 +1,27 @@
 package seedu.stockpal.commands;
 
+import seedu.stockpal.common.Messages;
 import seedu.stockpal.exceptions.StockPalException;
 
+
+import static seedu.stockpal.ui.Ui.printToScreen;
+
 public class HelpCommand extends Command {
     public static final String COMMAND_KEYWORD = "help";
     public static final String COMMAND_USAGE = COMMAND_KEYWORD + ": ";
 
     @Override
     public void execute() throws StockPalException {
-        super.execute();
+        printToScreen(Messages.HORIZONTAL_LINE
+                + "\n" + NewCommand.COMMAND_USAGE
+                + "\n\n" + EditCommand.COMMAND_USAGE
+                + "\n\n" + DeleteCommand.COMMAND_USAGE
+                + "\n\n" + InflowCommand.COMMAND_USAGE
+                + "\n\n" + OutflowCommand.COMMAND_USAGE
+                + "\n\n" + ListCommand.COMMAND_USAGE
+                + "\n\n" + ExitCommand.COMMAND_USAGE
+                + "\n" + Messages.HORIZONTAL_LINE

```

Comment:
May I clarify if 1 `\n` is intended for line 16 and 2 `\n` are intended for lines 17 - 23?

Reply:
Can consider using `System.lineSeparator()` (instead of `\n`) for cross platform compatability